deploy/helm/flightctl/templates/flightctl-db-deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
               value: {{ .Values.db.userPassword }}
             - name: POSTGRESQL_USER
               value: {{ .Values.db.user }}
+            - name: POSTGRESQL_MAX_CONNECTIONS
+              value: "200"
```

Comment:
should this be configurable?

Reply:
@gshilin-sdb can we make this a parameter of Values.db.max_connections and default that to 200 ?